l10n/en/footer-refresh.ftl

```diff
@@ -8,8 +8,7 @@ footer-refresh-advertise = Advertise with { -brand-name-mozilla }
 footer-refresh-firefox-release-notes = { -brand-name-firefox } Release Notes
 footer-refresh-mdn = MDN
 footer-refresh-follow-mozilla = Follow @{ -brand-name-mozilla }
-footer-refresh-x = X
-footer-refresh-x-formerly-twitter = X (formerly Twitter)
+footer-refresh-bluesky = Bluesky
```

Comment:
Sometimes it's here "Bluesky" literally, sometimes {-brand-name-bluesky} reference …

IIUC third party brands should no longer be added to the brand terms, so probably can be used spelled out fully in every single string(?)

Reply:
will check

Reply:
> Sometimes it's here "Bluesky" literally, sometimes {-brand-name-bluesky} reference 

this was a miss on my part, I meant to use the brand name ref consistently, still checking on the brand term standards

Reply:
As discussed internally, we're not going to include the 3rd party brand in FTL files (unless it's part of a larger sentence, in which case we should [use a comment](https://github.com/mozilla/bedrock/blob/6aafa46ea64613f1f74da27346e277de6e208117/l10n/en/download_button.ftl#L66) to indicate the words that should not be translated), will put directly in HTML